scripts/start.py

```diff
@@ -27,14 +27,16 @@
 import subprocess
 import time
 
-import python_utils
-
-from . import build
-from . import common
 from . import install_third_party_libs
 # Install third party libraries before importing other files.
```

Comment:
See my comment, please add more explanation (as I did) so other devs know why these need to be in the wrong order.